src/test/mockLocalMain.ts

```diff
@@ -0,0 +1,40 @@
+import 'jest-extended';
+
+export const ServiceContainer = { cradle: { localLogger: {child: jest.fn()}}};
+
+export const getServiceContainer = () => ServiceContainer;
```

Comment:
What do you think about making `getServiceContainer` and `workerFork` mock functions (`jest.fn()`) and then making some assertions that they were called correctly?

Reply:
I added workerFork in here; getServiceContainer needs to be a custom function so it can contain the cradle / logger info.